ckanext/xroad_integration/commands/xroad.py

```diff
@@ -49,6 +49,19 @@ def fetch_stats(ctx, config, days):
     utils.fetch_stats(days)
 
 
+@xroad_commands.command(
+    u'fetch_distinct_service_stats',
+    help='Fetches X-Road distinct service stats from catalog lister'
+)
+@click_config_option
```

Comment:
this is no longer needed

Reply:
Deprecated code, so I will not change anything here